iphone/Classes/GeolocationModule.m

```diff
@@ -494,19 +501,23 @@ - (void)performGeo:(NSString *)direction address:(NSString *)address callback:(G
   [callback start:params];
 }
 
-- (void)reverseGeocoder:(double)latitude longitude:(double)longitude withCallback:(JSValue *)callback
+- (JSValue *)reverseGeocoder:(double)latitude longitude:(double)longitude withCallback:(JSValue *)callback
 {
 #ifndef __clang_analyzer__ // Ignore static analyzer error here, memory will be released. See TIMOB-19444
-  ReverseGeoCallback *rcb = [[ReverseGeoCallback alloc] initWithCallback:callback];
+  KrollPromise *promise = [[KrollPromise alloc] initInContext:[self currentContext]];
```

Comment:
It will leak memory here. I think it should be -
```suggestion
  KrollPromise *promise = [[[KrollPromise alloc] initInContext:[self currentContext]] autorelease];
```